### DISCUSSION TITLE
How to use this icons in a React component?

### DISCUSSION THREAD

**Discussion by YuriNasci** [atomiclabs/cryptocurrency-icons#493] [Q&A]
I'm newbie in React, I just need a simple exemple.

**Comment by muhammedmed** [comment]


**Reply by Zio-4** [reply]
What does react icons have to do with this crypto icon pack? 

If I import like 

import UsdIcon from '../../../node_modules/cryptocurrency-icons/svg/white/usd.svg'

I get an error. Can you provide an example with this specific icon pack?
